src/AppBundle/Form/ActivityType.php

```diff
@@ -57,7 +116,7 @@ public function configureOptions(OptionsResolver $resolver)
     {
         $resolver->setDefaults(
             array(
-            'data_class' => 'AppBundle\Entity\Activity'
+                'data_class' => 'AppBundle\Entity\Activity'
```

Comment:
tout est vraiment indenté (alt + i) puis phpcbf de code sniffer ?

Reply:
of course sniff sniff is good, par contre il y avait bien un problème d'indentation ;)